ci/Makefile

```diff
@@ -13,18 +13,10 @@ CI_IMAGE = gcr.io/$(PROJECT)/ci:$(shell docker images -q --no-trunc ci | sed -e
 PYTHONPATH := $${PYTHONPATH:+$${PYTHONPATH}:}
 PYTHON := PYTHONPATH=$(PYTHONPATH)../batch:../hail/python:../gear:../web_common python3
 
-PY_FILES := $(shell find ci -iname \*.py -not -exec git check-ignore -q {} \; -print)
-
-flake8-stmp: $(PY_FILES)
-	python3 -m flake8 ci
-	touch $@
-
-pylint-stmp: $(PY_FILES)
-	$(PYTHON) -m pylint --rcfile ../pylintrc ci --score=n
-	touch $@
-
 .PHONY: check
-check: flake8-stmp pylint-stmp
+check:
+	$(PYTHON) -m flake8 ci
+	$(PYTHON) -m pylint --rcfile ../pylintrc ci --score=n
```

Comment:
Want to check that we don't want to `touch $@` and cleanup `rm -f check`, e.g use the file system to checkpoint the make step. This actually seems better, but I may not understand the benefit.

Reply:
No, we don't want.

check is quite old, and when it was added there were local deploy steps that were dependent on check. In order to make (re)deploy fast (ha!), we had check touch a stamp file.  I don't think this is relevant anymore because of many changes to the deployment logic.

For using a stamp file to be correct, we need the check step to depend on all the files that flake8 and pylint look at during check.  This includes all dependent modules.  These dependencies weren't being tracked correctly, so check was sometimes not being rerun as needed.

Since the caching benefit doesn't seem relevant anymore, instead of trying to track the dependencies correctly (and error prone endeavor given that it's manual), I just ripped it out.

Make sense?